### PR TITLE
CLC-5427 Add new feed source for Alerts / Release Notes

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.ets.berkeley.edu/jira/browse/CLC-5427

I got rid of two obsolete feature flags before adding the new one.

After the transition to the new feed sources, we'll want to delete the old "Alerts" class.

### CODE REVIEW

**Comment by raydavis** [comment]
Even if it miraculously looks OK to you, please don't merge this PR before tomorrow morning. I just wanted to get it up early for review / CI testing.

File: src/assets/javascripts/angular/controllers/pages/splashController.js [hunk 1]
```diff
@@ -9,14 +9,9 @@
...
-        $scope.splashNote = {
-          date: $filter('date')(data.alert.timestamp.epoch * 1000, 'MMM dd'),
-          link: data.alert.url,
-          snippet: data.alert.teaser,
-          title: data.alert.title
-        };
+        $scope.splashNote = data.alert;
       } else {
-        $scope.splashNote = data.entries[0];
+        $scope.splashNote = data.release_note;
```
Comment (christianv) [inline src/assets/javascripts/angular/controllers/pages/splashController.js]: @raydavis can we change this to `releaseNote`?

Comment (raydavis) [inline src/assets/javascripts/angular/controllers/pages/splashController.js]: I agree with you and JSHint on that, @christianv -- amended PR on its way...

**Comment by paulkerschen** [comment]
I agree with the happy half of Travis.

**Pull request merged**